modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -18,13 +18,14 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import {PROJECT_COORDINATE_SYSTEM} from './constants';
+
+const CONSTANTS = Object.keys(PROJECT_COORDINATE_SYSTEM)
+  .map(k => `const float COORDINATE_SYSTEM_${k} = ${PROJECT_COORDINATE_SYSTEM[k]}.;`)
```

Comment:
Nit: `k` => `key`

Reply:
+1. Nice touch!

Since this code is not as obvious to read as the verbatim declarations, please make variable name more descriptive

`COORDINATE_SYSTEM_GLSL_CONSTANTS`

Add comment explaining that we are generating GLSL constants from JS.